graphql-dgs-subscriptions-sse/src/main/kotlin/com/netflix/graphql/dgs/subscriptions/sse/DgsSSESubscriptionHandler.kt

```diff
@@ -51,7 +51,7 @@ import com.netflix.graphql.types.subscription.Error as SseError
 @RestController
 open class DgsSSESubscriptionHandler(open val dgsQueryExecutor: DgsQueryExecutor) {
 
-    @GetMapping("/subscriptions", produces = [MediaType.TEXT_EVENT_STREAM_VALUE])
+    @GetMapping("\${dgs.graphql.sse.path:/subscriptions}", produces = [MediaType.TEXT_EVENT_STREAM_VALUE])
```

Comment:
Does this work if `dgs.graphql.sse.path` is not explicitly set?

Reply:
Yeah so if you don't have the property in your application.yml file it will default to /subscriptions (as was the behaviour before)